tests/test_both.py

```diff
@@ -0,0 +1,33 @@
+from dacite import from_dict
```

Comment:
Can you quickly elaborate how these tests run both in mocked and live mode?

Reply:
This fixture is run twice, once emitting the mock client and once emitting the live client:

https://github.com/mamba-org/quetz-client/blob/1180ccfa001b0ed3e894f6c4216ab2fd31cee31e/tests/conftest.py#L125-129